document/CoreFormat.mdk

```diff
@@ -1891,6 +1989,23 @@ a stable, opaque identifier for the rule.
 `"CA2101"`
 ~
 
+~ IsoNote
+Rule identifiers must be stable for two reasons:
+
+1. So specific checks can be disabled in build automation scripts, without the risks
+of the scripts breaking if a rule id changes.
+2. So result management systems can compare results from one run to the next,
+without erroneously designating results as "new" because a rule id has changed.
+
+Rule identifiers should be opaque -- that is, they should not convey information to
+a user -- because a rule's implementation might change over time.
```

Comment:
Another common source of rule id instability has to do with refactoring. i.e. the collapse of checks into things that address a common weakness or the reverse, exploding a check that was too general.